bin/stacks/routing-dashboard-stack.ts

```diff
@@ -76,11 +85,40 @@ export class RoutingDashboardStack extends cdk.NestedStack {
       },
     })
 
+    const poolCacheLambdaWidgetArray: LambdaWidget[] = []
+    poolCacheLambdaNameArray.forEach((poolCacheLambdaName) => {
```

Comment:
afaict this creates multiple widgets instead of creating one with metric (time series data) for each chain, right? feel like the latter is cleaner (and is also what Pote originally suggested)

Reply:
Oh I'm not sure how to do that, can you give a suggestion? I'm also looking

Reply:
think i figured it out

Reply:
cool, I was thinking adding 
```
            ['AWS/Lambda', 'Errors', 'FunctionName', poolCacheLambdaName],
            ['.', 'Invocations', '.', '.'],
```
for each `poolCacheLambdaName` in `poolCacheLambdaNameArray` to the `metrics` array field

Reply:
Ok updated it